docs/howto/openstack/octavia/lbaas-tcp.md

```diff
@@ -0,0 +1,742 @@
+    | id                  | 69457661-197e-444a-b9ce-54394263dafa |
+    | name                | 198.51.100.129                       |
+    | port_details        | None                                 |
+    | port_id             | None                                 |
+    | project_id          | dfc700467396428bacba4376e72cc3e9     |
+    | qos_policy_id       | None                                 |
+    | revision_number     | 0                                    |
+    | router_id           | None                                 |
+    | status              | DOWN                                 |
+    | subnet_id           | None                                 |
+    | tags                | []                                   |
+    | updated_at          | 2023-01-16T20:25:25Z                 |
+    +---------------------+--------------------------------------+
+    ```
+
+    Then, load the ID of the new floating IP to a variable:
+
+    ```bash
+    FLOAT_IP_ID=$(openstack floating ip list \
+        --floating-ip-address 198.51.100.129 -c ID -f value)
+    ```
+
+    Do the same for the VIP port ID of your load balancer:
+
+    ```bash
+    VIP_PORT_ID=$(openstack loadbalancer show mylb \
+        -c vip_port_id -f value)
+    ```
+
+    Finally, assign the floating IP to your load balancer like so:
+
+    ```bash
+    openstack floating ip set --port $VIP_PORT_ID $FLOAT_IP_ID
+    ```
+
+## Testing the load balancer
+
+Each of the two test servers runs `ncat` to continuously listen
+on port 61234/TCP and respond to clients with a simple message,
+revealing its hostname:
+
+```bash
+ncat -kV -l 61234 -c 'echo Yello from $(hostname)!'
+```
+
+Since the load balancer has a floating IP (`198.51.100.129`,
+in our example). and we know the port it listens to, we can
+try connecting to it via `wget` and see what happens:
+
+```bash
+wget -q 198.51.100.129:61234 -O -
+```
+
+```plain
+Yello from srv-lbaas-1!
+```
+
+It looks like we talked to the first test server. If we run the
+exact same `wget` command for a second time, since the load balancer
+distributes client connection requests in a round-robin fashion,
+we expect to talk to the second server:
+
+```bash
+wget -q 198.51.100.129:61234 -O -
+```
+
+```plain
+Yello from srv-lbaas-2!
+```
+
+During our testing, executing the `wget` command repeatedly, we were
+getting those two hostnames one after the other --- and that was
+proof our TCP load balancer was working as expected. But there is
+one more expectation of any load balancer; the ability to skip
+backend hosts when they are inaccessible.
+
+To test our load balancer in this new scenario, let us first use
+`wget` to connect to port 61234 and jot down the backend server
+that will respond:
+
+```bash
+wget -q 198.51.100.129:61234 -O -
+```
+
+```plain
+Yello from srv-lbaas-2!
+```
+
+We see that `srv-lbaas-2` responded. That means the next time we
+try to connect, `srv-lbaas-1` will respond. But if we SSH into
```

Comment:
Residual reference to SSH.

Reply:
No, it is not, I am just describing a course of action.